slf4j-api/src/main/java/org/slf4j/helpers/Util.java

```diff
@@ -53,7 +53,7 @@ public static boolean safeGetBooleanSystemProperty(String key) {
         if (value == null)
             return false;
         else
-            return value.equalsIgnoreCase("true");
```

Comment:
What about just return `Boolean.parseBoolean(value)` instead?

Reply:
That's indeed even better in this specific case; pushed just now.

The previous iteration was made by applying [the "Equals avoids null" recipe](https://docs.openrewrite.org/reference/recipes/java/cleanup/equalsavoidsnull); Perhaps a subsequent recipe can convert that into the `Boolean.parseBoolean` calls in other places as well.